backend/pricing/pricing.go

```diff
@@ -651,9 +668,21 @@ func (w *Worker) reportUsage(ctx context.Context, workspaceID int, productType *
 	}
 
 	invoiceLines := map[model.PricingProductType]*stripe.InvoiceLine{}
-	for _, line := range invoice.Lines.Data {
+	for _, line := range lo.FindUniquesBy(invoice.Lines.Data, func(item *stripe.InvoiceLine) string {
+		return item.InvoiceItem
+	}) {
 		productType, _, _, _, _ := GetProductMetadata(line.Price)
 		if productType != nil {
+			if invoiceLines[*productType] != nil {
```

Comment:
`invoiceLines[*productType] != nil` wouldn't be true until it's set on 686 right?

Reply:
correct, this deletes duplicate line items for old prices for a given product